crates/experimental-api/src/exe/err.rs

```diff
@@ -56,6 +74,14 @@ impl From<blockifier::transaction::errors::TransactionExecutionError>
     }
 }
 
+impl From<cairo_vm::types::errors::program_errors::ProgramError> for Error {
+    fn from(
+        error: cairo_vm::types::errors::program_errors::ProgramError,
+    ) -> Self {
+        Self::Program(error)
+    }
+}
+
 impl From<iamgroot::jsonrpc::Error> for Error {
     fn from(error: iamgroot::jsonrpc::Error) -> Self {
         Self::IamGroot(error)
```

Comment:
`#[from]` annotations should avoid the need for explicit `From` implementation

Reply:
It actually forks for on all but `IamGroot` one :)